### ISSUE TITLE
[1.7.10] Crash to desktop when mining ores.

### ISSUE THREAD

**Issue by Xireka** [TechReborn/TechReborn#574]
When mining ores the game crashes to desktop, until now this only happened with Techreborn's silver and lead ores (didn't try the others).

As soon as the block would break the game crashes, reloading it works fine and the block that crashed is mined and inside my inventory.

Crash report 1 with silver ore:
http://pastebin.com/93e7VZgF

Crash report 2 with lead ore:
http://pastebin.com/D612HJ0R

Thanks in advance for any help, I really like your mod and it would be a shame to play without it :(

**Comment by modmuss50** [comment]
at com.rwtema.extrautils.tileentity.generators.BlockGenerator.func_149691_a(BlockGenerator.java:192)

**Comment by Xireka** [comment]
I know, yet it happens when mining ores and not anything generator related, also I have no generator placed in my world or near me when mining (deep in a cave). :(

I also opened an issue over at Extra utilities on this.

I just wanted to report it since I noticed other ores work just fine.

I noticed a new update was made to techreborn for 1.7.10 in May that I didn't have, I'll try seeing if the crash still happens with that version.
